source/administration/command-line-tools.rst

```diff
@@ -112,6 +113,23 @@ platform channel add
 
       sudo ./platform channel add mychannel user@example.com username
 
+platform channel archive
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+  Description
+    Archive a channel. To restore a channel from the archive, see `platform channel restore`_.
```

Comment:
Should we maybe define what archiving means if we don't do it elsewhere? Something like "Archive a channel. Archived channels won't be accessible by users, but they will remain in the database. To restore..."